packages/reactotron-core-server/src/reactotron-core-server.ts

```diff
@@ -112,17 +119,20 @@ export default class Server {
       this.emitter.emit("connect", partialConnection)
 
       // when this client disconnects
-      socket.on("disconnect", () => {
+      socket.on("close", () => {
```

Comment:
`ws` doesn't have a "disconnect" event. Just `close`. So lets do that instead.

Reply:
oops

Reply:
Oops.